src/dashboard/src/App.tsx

```diff
@@ -81,7 +81,7 @@ const Layout: React.FC<RouteComponentProps> = ({ location, history }) => {
               <Route path="/submission" component={Submission}/>
               <Route path="/jobs/:cluster" component={Jobs}/>
               <Route path="/jobs" component={Jobs}/>
-              <Route path="/job/:team/:clusterId/:jobId" component={Job}/>
+              <Route path="/job/:team/:clusterId/:jobId/:jobPriority?" component={Job}/>
```

Comment:
Are you renaming single job route `/job/:team/:clusterId/:jobId` to `/job/:team/:clusterId/:jobId/:jobPriority` for transferring priority variable?